client/src/modules/home/plan-card/plan-card.js

```diff
@@ -6,7 +6,7 @@ import './plan-card.css';
 
 class PlanCardController {
   constructor() {
-    this. handleDeleteClick = () => {
+    this.handleDeleteClick = () => {

```

Comment:
can we use the same syntax model you used in ideas-card here? declare outside constructor and bind inside